publication-event-handlers/src/main/java/no/unit/nva/publication/events/handlers/dynamodbstream/DynamodbStreamToEventBridgeHandler.java

```diff
@@ -19,25 +28,35 @@
  * <p>Notice a DynamoDB stream can only have two streams attached before we it can lead into throttling and performance
  * issues with DynamodDB, this is why we have this handler to publish it to EventBridge.
  */
-public class DynamodbStreamToEventBridgeHandler implements RequestHandler<DynamodbEvent, Void> {
+public class DynamodbStreamToEventBridgeHandler implements RequestHandler<DynamodbEvent, URI> {
 
     public static final String AWS_REGION = "AWS_REGION";
     public static final String DYNAMODB_UPDATE_EVENT_TOPIC = "PublicationService.Database.Update";
+    public static final String EVENTS_BUCKET = new Environment().readEnv("EVENTS_BUCKET");
+    public static final String DYNAMO_EVENTS_FOLDER_IN_BUCKET = "dynamoEvents";
     private final EventPublisher eventPublisher;
+    private final S3Client s3Client;
 
     @JacocoGenerated
     public DynamodbStreamToEventBridgeHandler() {
-        this(defaultEventBridgePublisher());
+        this(defaultS3Client(), defaultEventBridgePublisher());
     }
 
-    protected DynamodbStreamToEventBridgeHandler(EventPublisher eventPublisher) {
+    protected DynamodbStreamToEventBridgeHandler(S3Client s3Client, EventPublisher eventPublisher) {
         this.eventPublisher = eventPublisher;
+        this.s3Client = s3Client;
     }
 
     @Override
-    public Void handleRequest(DynamodbEvent event, Context context) {
+    public URI handleRequest(DynamodbEvent event, Context context) {
+        var savedFile = storeEventInS3(event);
```

Comment:
try to save the file but do not fail if cannot save the file. (Preserve current functionality)